donate/settings.py

```diff
@@ -536,9 +538,10 @@
         'NAME': 'django.contrib.auth.password_validation.NumericPasswordValidator',
     },
 ]
-AUTHENTICATION_BACKENDS = (
-    'mozilla_django_oidc.auth.OIDCAuthenticationBackend',
-)
+
+# AUTHENTICATION_BACKENDS = (
+#     'mozilla_django_oidc.auth.OIDCAuthenticationBackend',
+# )
```

Comment:
Is this just for review?

Reply:
yeah - @cadecairos's updating his settings PR, which also fixes this but "properly".